ui/src/index.tsx

```diff
@@ -208,6 +209,10 @@ class Root extends PureComponent {
                               path=":id/export"
                               component={VariableExportOverlay}
                             />
+                            <Route
+                              path="new"
```

Comment:
I haven’t checked but I am wondering if “new” is consistent with the other Create overlays

Reply:
seems to be!